azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/VerificationIPFlow.java

```diff
@@ -0,0 +1,197 @@
+             */
+            WithProtocol withDirection(Direction direction);
+
+            /**
+             * Set inbound direction.
+             *
+             * @return the next stage of the definition.
+             */
+            WithProtocol inbound();
+
+            /**
+             * Set outbound direction.
+             *
+             * @return the next stage of the definition.
+             */
+            WithProtocol outbound();
+        }
+
+        /**
+         * Protocol to be verified on. Possible values include: 'TCP', 'UDP'.
+         */
+        interface WithProtocol extends HasProtocol.DefinitionStages.WithProtocol<WithLocalIP, Protocol> {
+            /**
+             * Set TCP protocol.
+             *
+             * @return the next stage of the definition
+             */
+            WithLocalIP withTCP();
+
+            /**
+             * Set UDP protocol.
+             *
+             * @return the next stage of the definition
+             */
+            WithLocalIP withUDP();
+        }
+
+        /**
+         * The local IP address. Acceptable values are valid IPv4 addresses.
+         */
+        interface WithLocalIP {
+            /**
+             * Set the localIPAddress value.
+             *
+             * @param localIPAddress the localIPAddress value to set
+             * @return the next stage of the definition.
+             */
+            WithRemoteIP withLocalIPAddress(String localIPAddress);
+        }
+
+        /**
+         * The remote IP address. Acceptable values are valid IPv4 addresses.
+         */
+        interface WithRemoteIP {
+            /**
+             * Set the remoteIPAddress value.
+             *
+             * @param remoteIPAddress the remoteIPAddress value to set
+             * @return the next stage of the definition.
+             */
+            WithLocalPort withRemoteIPAddress(String remoteIPAddress);
+        }
+
+        /**
+         * The local port. Acceptable values are a single integer in the range
+         * (0-65535). Support for * for the source port, which depends on the
+         * direction.
+         */
+        interface WithLocalPort {
+            /**
+             * Set the localPort value.
+             *
+             * @param localPort the localPort value to set
+             * @return the next stage of the definition.
+             */
+            WithRemotePort withLocalPort(String localPort);
+        }
+
+        /**
+         * The remote port. Acceptable values are a single integer in the range
+         * (0-65535). Support for * for the source port, which depends on the
+         * direction.
+         */
+        interface WithRemotePort {
+            /**
+             * Set the remotePort value.
+             *
+             * @param remotePort the remotePort value to set
+             * @return the next stage of the definition.
+             */
+            WithExecute withRemotePort(String remotePort);
```

Comment:
Swagger spec says "Support for * for the source port, which depends on the
         * direction.", so I assume '*' is a possible value.